applicationset/webhook/testdata/gitlab-event.json

```diff
@@ -55,11 +55,11 @@
     },
     "repository": {
       "name": "name",
-      "url": "ssh://git@gitlab:2222/group/name.git",
+      "url": "ssh://git@gitlab.com:2222/group/name.git",
       "description": "",
-      "homepage": "https://gitlab/group/name",
-      "git_http_url": "https://gitlab/group/name.git",
-      "git_ssh_url": "ssh://git@gitlab:2222/group/name.git",
+      "homepage": "https://gitlab.com/group/name",
+      "git_http_url": "https://gitlab.com/group/name.git",
+      "git_ssh_url": "ssh://git@gitlab.com:2222/group/name.git",
```

Comment:
Why do these need to be changed?

Reply:
Not strictly necessary. Just making them consistent with the other providers’ test data, which all have TLDs on their hostnames.

If it’s a problem, I can revert.